pkg/controller/v1beta1/inferenceservice/components/predictor.go

```diff
@@ -76,9 +76,12 @@ func (p *Predictor) Reconcile(isvc *v1beta1.InferenceService) (ctrl.Result, erro
 	var sRuntimeAnnotations map[string]string
 
 	annotations := utils.Filter(isvc.Annotations, func(key string) bool {
-		return !utils.Includes(constants.ServiceAnnotationDisallowedList, key)
+		return !utils.Includes(p.inferenceServiceConfig.ServiceAnnotationDisallowedList, key)
 	})
 
+	fmt.Printf("\n Predictor custom annotations %v\n", p.inferenceServiceConfig.ServiceAnnotationDisallowedList)
+	fmt.Printf("\n Predictor custom labels %v\n", p.inferenceServiceConfig.ServiceLabelDisallowedList)
```

Comment:
Use `p.Log.Info` instead of `fmt.Printf`?

Reply:
Oh, it is a left over from my manual tests, I think we can keep it as debug.